package.json

```diff
@@ -38,7 +38,8 @@
   "devDependencies": {
     "@types/debug": "^4.1.5",
     "@types/jest": "^25.2.3",
-    "@types/node": "^10",
+    "@types/node": "^12",
+    "@types/babel__traverse": "7.0.4",
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/63431 - had to pin it to get it working locally. Haven't looked into it too much at the moment

Reply:
worth checking if you can get this running by removing this

Reply:
Can open another PR to bump packages and ultimately remove this